tests/test_training.py

```diff
@@ -49,8 +49,8 @@ def test_moe_training(max_steps: int = 100, threshold: float = 0.9, num_experts=
 
     all_expert_uids = [f'expert.{i}' for i in range(num_experts)]
     with background_server(expert_uids=all_expert_uids, device='cpu', optim_cls=SGD, hidden_dim=64, num_handlers=1) \
-            as (server_endpoint, dht_endpoint):
-        dht = DHT(start=True, initial_peers=[dht_endpoint])
+            as (server_endpoint, initial_peers):
```

Comment:
Returning `initial_peers` when creating a single server is a bit unexpected; maybe give it a name like `dht_multiaddr`?

Reply:
Renamed to `dht_maddrs` everywhere (unless it's named `_` because it's not used)

Reply:
But shouldn't a single DHT instance and its underlying p2p instance have specifically one multiaddr, not many?

Reply:
Got it for now, created https://github.com/learning-at-home/hivemind/issues/311 for the future